feat(admin): Add `/admin` page

TimDiekmann opened the pull request:
Currently, the `/admin` page can create new rooms. `Name` is the internal room name, which is used
by SocketIO (SIO uses strings as identifier). Label is the display name. Layout can be either an URL
or a provided layout name.

cc: #39

davidschlangen commented:
Is this mostly for testing? I would think that in deployment, one knows in advance which rooms are needed and would want to have them created via a config file or a start up script.

TimDiekmann commented:
I'm sorry I'm just answering now, somehow I must have overlooked your question...

Of course this is also suitable for testing. But I implemented this as an answer to #39. 

If you want to start a Slurk session, you have to configure it first. This is best illustrated by an example: 
I want to start a task online. I created a layout and implemented several plugins. This layout should of course be loaded without redirecting the turkeys. These files I upload somewhere where they are accessible from everywhere. I now start the session, go to *.../admin*, and create a new room. As layout I insert the URL from the file uploaded somewhere else. I can specify the room I just created during token generation, so bots or turkeys are routed to this room instead of the previously typical `Waiting Room`.

The page requires, like the token generation, the private key from config.ini.

TimDiekmann commented:
While a unified way may be nice for adding rooms, this is not on our priority list. I may add such functionality at a later point.
